### PR TITLE
Switch plugin for XJC generation

### PR DESCRIPTION
**Pull request by ljacomet**

### CODE REVIEW

File: xml/config/checkstyle-suppressions.xml [hunk 1]
```diff
@@ -5,6 +5,6 @@
     "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
 
 <suppressions>
-  <suppress files="src[\\/]generated[\\/]java[\\/]" checks=".*"/>
+  <suppress files="build[\\/]xjc[\\/]generated-sources[\\/]" checks=".*"/>
```
Comment (henri-tremblay) [inline xml/config/checkstyle-suppressions.xml]: I was wondering why we were using square braces everywhere. We actually do not need them.

Comment (ljacomet) [reply xml/config/checkstyle-suppressions.xml]: No idea, removing all of them from that file breaks checkstyle, so leaving them 😉

Comment (henri-tremblay) [reply xml/config/checkstyle-suppressions.xml]: Weird, worked for me. Anyway. If it works...

Comment (chrisdennis) [reply xml/config/checkstyle-suppressions.xml]: It's a regular expression thats why Louis has no idea what it's doing.  It's matching one of "/" or "\", for windows or *nix.

Comment (henri-tremblay) [reply xml/config/checkstyle-suppressions.xml]: Oh! Of course... Too much Java, I thought the backslash was escaped and then escaping the slash...

**Comment by henri-tremblay** [comment]
build might pass when rebased

**Pull request merged**